ditto/src/test/java/eu/xenit/testing/ditto/api/AlfrescoDataSetTest.java

```diff
@@ -11,7 +11,8 @@
     private AlfrescoDataSet dataSet = AlfrescoDataSet.bootstrapAlfresco()
             .skipToTransaction(12345L)
             .addTransaction(txn -> {
-                txn.addDocument("foo.txt", doc -> {
+                txn.addDocument(null, doc -> {
```

Comment:
just use addNode(Consumer<NodeCustomizer> customizer) instead

or do you need the `node.type(Content.CONTENT);` at DefaultTransaction:181?

Reply:
hmm, it's called a callback in this case.